tests/setup_test.go

```diff
@@ -7008,3 +7008,136 @@ func TestProjectCommitStatusRedelivery(t *testing.T) {
 		}
 	})
 }
+
+func TestRunRequiredEnvVariables(t *testing.T) {
+	t.Parallel()
+
+	config := `
+		{
+			runs: [
+			{
+				name: 'run01',
+				tasks: [
+				{
+					name: 'task01',
+					runtime: {
+					containers: [
+						{
+						image: 'alpine/git',
+						},
+					],
+					},
+					steps: [
+					  { type: 'run', command: 'env -u AGOLA_SSHPRIVKEY' },
```

Comment:
is `-u ...` really needed?

Reply:
yes because AGOLA_SSHPRIVKEY can contains value with equal characters and the ParseEnv split will fail

Reply:
I'm not sure about which is the issue but there're already other tests that use don't have such issue. Anyway it should be fixed in ParseEnvs

Reply:
I are investigating well, in the other tests we get the log of directruns and I see they have AGOLA_SSHPRIVKEY variable with empty value, so they work.
In my test the AGOLA_SSHPRIVKEY is a multiline variable, so I think the best way is to skip this variable with -u